netlify/functions/util/auth.js

```diff
@@ -0,0 +1,58 @@
+const { AuthorizationCode } = require("simple-oauth2");
+const querystring = require("querystring");
+
+const config = require("./config");
+
+const base =
+  config.context === "production" || config.context === "deploy-preview"
+    ? "https://auth.esp.vmware.com/api/auth/v1"
```

Comment:
Use environment variables?

Reply:
for the auth endpoints? i didn't bother because when they get swapped to csp it'll be a bit of a rewrite

Reply:
Ah got it. These are different from the lookup service URLs...